src/Game/Formatter/WeatherFormatter.php

```diff
@@ -9,64 +9,114 @@
 class WeatherFormatter
 {
 
+
+    /**
+     * @param $weightedValues
+     *
+     * @return string
+     *
+     * Utility function for getting random values with weighting.
+     * Pass in an associative array, such as array('A'=>5, 'B'=>45, 'C'=>50)
+     * An array like this means that "A" has a 5% chance of being selected, "B" 45%, and "C" 50%.
+    */
+
+    public static function getWeather(array $weightedValues) {
+
+        $rand = mt_rand(1, (int) array_sum($weightedValues));
+
+        foreach ($weightedValues as $key => $value) {
+
+            $rand -= $value;
+            if ($rand <= 0) {
+
+                return $key;
+
+            }
+        }
+    }
+
     /**
      * @param $game
      *
      * @return string
      */
+
     public static function format(Game $game)
     {
         // Send message according to GameState and weather
-        $state= $game->state;
-
-        if($state == GameState::FIRST_NIGHT){
+        $state = $game->state;
+        /** 
+        * Create an associate array with the value as weight.
+        * To modify the weight, just change the value with the integer.
+        * Raining = 25%, cloudy = 50%, sunny= 25%  
+        */
+        $weight = array(
+            "rainy" => 25, 
+            "cloudy" => 50, 
+            "sunny" => 25,
+            );
+
+        $Weather = WeatherFormatter::getWeather($weight);
+        $messages = array(":rain_cloud:",":cloud:",":sunny:");
+
+        if ($state == GameState::FIRST_NIGHT){
+
+            if (file_exists("src/Game/Data/first_weather.txt")){
+                $messages = file("src/Game/Data/first_weather.txt");
+            }
 
-            if($game->weather == 1){
-                $msg = ":moon: :rain_cloud: The rain comes down in torrents as the village sleeps, unaware of the horror that lurks outside in the wet. It is the middle of the night.";
+            if ($Weather == "rainy"){
+                $msg = $messages[0];
             }
 
-            else if($game->weather==1){
-                $msg = ":moon: :cloud: The cloud covered the sky blocking even the few glimmering light. ";
+            else if ($Weather == "cloudy"){
+                $msg = $messages[1];
             }
 
             else{
-                $msg = ":moon: The village sleeps, unaware of the horror that lurks outside in the dark. It is the middle of the night.";
+                $msg = $messages[2];
             }
 
         }
 
-        if($state == GameState::DAY){
+        if ($state == GameState::DAY){
+            if (file_exists("src/Game/Data/day_weather.txt")){
```

Comment:
consider having these files load as part of the application boot process. this feels a little hidden away and maybe a little sneaky to me.

maybe encapsulate weather loading and retrieval in a Weatherman class and supply a new instance of it to the Game constructor, for example.